Adding Portal (comm) port to Omada-Controller

StevenMcElligott opened the pull request:
**Description**
First ever PR so be really nice lol. This chart is missing HTTPS port 8843 to access the portal and has been broken/incubator for a while. I modified questions.yaml and charts.yaml to add in the missing port similar to unifi's docker controller. 
Fixes # <!--(issue)-->

**Type of change**

- [ ] Feature/App addition
- [ ] Bugfix
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor of current code

**How Has This Been Tested?**
Ran these fixes inside the Dev container and tested OK

  accepts "Dynamic/Tpl" additionalContainers                      PASS (0.16s)
  accepts static additionalContainers                             PASS (0.15s)

Finished in 21.97995s
140 tests, 295 assertions, 0 failures, 0 errors, 0 skips

**Notes:**
First PR so if I'm missing anything or suck let me know

**Checklist:**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests to this description that prove my fix is effective or that my feature works
- [ ] I increased versions for any altered app according to semantic versioning

StevenMcElligott committed:
Update values.yaml

Adding missing ports for Omada to work, let me know how off I am lol

StevenMcElligott committed:
Adding ports for Omada-Controller

Adding the missing port to Omada-Controller inside questions.yaml and values.yaml

stavros-k commented:
Can you also please 
- update `docs/manual/default-ports.md` (also put the warning about conflicting ports with unifi!
- update to HTTPS as default in `questions.yaml` for main service
- bump minor version in `Chart.yaml`

Thanks!

StevenMcElligott committed:
Merge branch 'master' into StevenMcElligott-Omadapatch

StevenMcElligott committed:
Adding ports for Omada-Controller

Adding the missing port to Omada-Controller inside questions.yaml and values.yaml, updated ports in manual and changed version

StevenMcElligott reviewed (commented):
Sounds good, changes made Stravos!

stavros-k commented:
> Sounds good, changes made Stravos!

Awesome.

Just a funny note: It's Stavros, stravos means blind in greek lol 😂😂.

I get that a lot, so don't worry :P

stavros-k reviewed (approved):
LGTM

Ornias1993 commented:
> Just a funny note: It's Stavros, stravos means blind in greek lol 😂😂.


Maybe I really need to review this again in that case ;-)

stavros-k commented:
@all-contributors please add @StevenMcElligott for code
